plugins/dotnet-db-sqlserver.yml

```diff
@@ -0,0 +1,12 @@
+id: dotnet-db-sqlserver
+name: MS SQL Server DB
+description: Use an MS SQL Server database in the service generated by Amplication
+repo: https://github.com/amplication/plugins/tree/master/plugins/dotnet-db-sqlserver
+npm: '@amplication/plugin-dotnet-db-sqlserver'
+icon: db-mssql.png
+github: https://github.com/amplication/plugins/tree/master/plugins/dotnet-db-sqlserver
+website: https://github.com/amplication/plugins/tree/master/plugins/dotnet-db-sqlserver
+type: official
+generator: [generator-dotnet-webapi]
```

Comment:
Is that the indication for .NET plugins? Did we add it to the NestJS plugins?

Reply:
yep, the generator will show which generator are compatible with a plugin